README.md

```diff
@@ -78,10 +79,6 @@ A list of awesome companies currently using Ruby as one of their main backend la
 - [Cleo](https://web.meetcleo.com/) - The AI assistant that's helped 6 million people avoid their overdraft, build credit, and budget better.
 - [Maybe](https://maybe.co/) - Open-source OS for your personal finances.
 
-## Private Members' Clubs
-
-- [Soho House](https://sohohouse.com/) - A club for creatives.
```

Comment:
you got beef with soho ? :D

Reply:
huh how did that happen?? i'm sorry. will fix.

Reply:
Fixed! Can't believe I fat-finger-nuked Soho house 🤣